src/stampy_nlp/routes.py

```diff
@@ -126,7 +127,7 @@ def search_api():
     query = request.args.get('query', DEFAULT_QUERY)
     top_k = as_int('top', DEFAULT_TOPK)
     status = request.args.getlist('status')
-    show_live = as_bool('showLive', 'true')
+    show_live = as_bool('showLive', str(not status))
```

Comment:
(minor) not related to changes here, but IMHO single source of truth for `status` would be nicer than `show_live`, something like following (but with correct string / enum value):

```py
status = request.args.getlist('status') or ['live']
```

Reply:
agreed. The only problem being that there are already various consumers of this endpoint :/